### PR TITLE
postproc: Print last line in case of error

### PR DESCRIPTION
**Pull request by Ram-Z**
When a postproc fails, provide last line in addition to exit code.

before:
image

after:
image

### CODE REVIEW

**Comment by thezoggy** [comment]
to note that the things you see with the T() is stuff thats translated, so changing that line like that would invalidate the translation. although im sure we could prob cleanup the translation to make a little more sense with the line, `Script (exit) lastline: (#) ####`

**Comment by Ram-Z** [comment]
Ah. Thanks. I hadn't given that T() much thought.                        
                                                                         
I went with consistency with the string in the popup when clicking on a  
download.                                                                
                                                                         
There's not necessarily much screen estate in that field, so I'd rather  
avoid unnecessary words.                                                 
                                                                         
Does "Exit" in this context need translating?

**Comment by Ram-Z** [comment]
OK. Here's an updated patch.

I first made the `Exit(%s) %s` string in the popup translatable. This should not introduce any changes besides adding a new translatable text.

Then I reuse that new string in the main history view.

File: sabnzbd/postproc.py [hunk 1]
```diff
@@ -577,18 +577,18 @@ def process_job(nzo: NzbObject):
...
                 nzo.set_unpack_info(
                     "Script",
-                    '%s%s <a href="./scriptlog?name=%s">(%s)</a>' % (script_ret, script_line, script_output, T("More")),
+                    '%s <a href="./scriptlog?name=%s">(%s)</a>' % (script_msg, script_output, T("More")),
                     unique=True,
                 )
             else:
                 # No '(more)' button needed
-                nzo.set_unpack_info("Script", "%s%s " % (script_ret, script_line), unique=True)
+                nzo.set_unpack_info("Script", "%s " % script_msg, unique=True)
```
Comment (Ram-Z) [inline sabnzbd/postproc.py:591]: I kept this extra space in ``%s "` as I wasn't sure if it was needed for anything.

**Comment by Safihre** [comment]
I'm working on some improvements for the post processing scripts and will include this improvement. So I haven't forgotten this PR :)

**Comment by Ram-Z** [comment]
Great, thanks.

**Pull request closed without merging**